javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,19 @@
+var $text = document.querySelectorAll('span');
+var index = 0;
+
+window.addEventListener('keydown', function (event) {
+
+  var $currentChar = $text.item(index).textContent;
+
+  if ($currentChar === event.key) {
+    $text.item(index).className = 'spanText correct';
```

Comment:
What is the `.item` method doing here? Is there another way you could accomplish this?

Reply:
<img width="1233" alt="Screen Shot 2021-03-30 at 11 32 02 AM" src="https://user-images.githubusercontent.com/78828085/113015834-ea098800-914b-11eb-8123-a4832462b186.png">
removed .item method and put $text[index]